guides/hack/03-expressions-and-operators/01-introduction.md

```diff
@@ -1,29 +1,46 @@
-**Topics covered in this section**
-
-* [Arithmetic, `+`, `*` etc](addition.md)
-* [Assignment, `=`, `+=` etc](assignment.md)
-* [await](await.md)
-* [Bitwise Operators](bitwise-operators.md)
-* [Casting, `(int)`, `(float)`, `(bool)` `(string)`](casting.md)
-* [Coalesce, `??`](coalesce.md)
-* [Comparisons, `>` etc](comparisons.md)
-* [Decrement, `--`](incrementing-and-decrementing.md)
-* [echo intrinsic function](echo.md)
-* [Equality operators](equality.md)
-* [Error control, `@`](error-control.md)
-* [exit intrinsic function](exit.md)
-* [Increment, `++`](incrementing-and-decrementing.md)
-* [invariant intrinsic function](invariant.md)
-* [list intrinsic function](list.md)
-* [Logical operators, `&&`, `||`, `!`](logical-operators.md)
-* [Member selection, `->`, `?->`](member-selection.md)
-* [new](new.md)
-* [Operator precedence](operator-precedence.md)
-* [Pipe, `|>`](pipe.md)
-* [Scope resolution, `::`](scope-resolution.md)
-* [Spaceship, `<=>`](equality.md)
-* [String concatenation, `.`](string-concatenation.md)
-* [Subscripting, `[]`](subscript.md)
-* [Ternary, `?` `:`](ternary.md)
-* [Type Assertions `is`, `as`, `?as`](type-assertions.md)
-* [Yield](yield.md)
+When combined, operators evaluate according to their associativity. For more information, see [Operator Precedence](/hack/expressions-and-operators/operator-precedence).
+
+## Assignment Operators
+* [Assignment](/hack/expressions-and-operators/assignment) (`=`, `+=`, and more)
+* [Coalescing Assignment](/hack/expressions-and-operators/coalesce#coalescing-assignment-operator) (`??=`)
+
+## Comparison Operators
+* [Comparison](/hack/expressions-and-operators/comparisons) (`>`, `>=`,`<`,`<=`)
+* [Equality Comparison](/hack/expressions-and-operators/equality) (`==`, `!=`, `===`,`!==`)
+* [Logical Comparison](/hack/expressions-and-operators/logical-operators) (`&&`, `||`, `!`)
+* [Spaceship Comparator](/hack/expressions-and-operators/equality#the-spaceship-operator) (`<=>`)
+
+## Arithmetic Operators
+* [Arithmetic](/hack/expressions-and-operators/arithmetic) (`+`, `-`, `*`, `/`, and more)
+* [Increment](/hack/expressions-and-operators/incrementing-and-decrementing) (`++`)
+* [Decrement](/hack/expressions-and-operators/incrementing-and-decrementing) (`--`)
+
+## Bitwise and Bit Shift Operators
+* [Bitwise](/hack/expressions-and-operators/bitwise-operators) (`&`, `|`, `^`, `<<`, `>>`, `~`)
+
+## Class and Member Operators
+* [Create an Object](/hack/expressions-and-operators/new) (`new`)
+* [Access Instance Members](/hack/expressions-and-operators/member-selection) (`->`, `?->`)
```

Comment:
I think 'member' is a term best avoided FWIW. PHP uses the term, but it's sufficiently vague that it can cause confusion. 'methods and properties' means the same thing, and you can often specifically say 'methods' or 'properties'.

Reply:
You're right. I felt weird including it here and I should have stuck to my instinct.